lms/envs/aws.py

```diff
@@ -795,3 +795,6 @@
 # The extended StudentModule history table
 if FEATURES.get('ENABLE_CSMH_EXTENDED'):
     INSTALLED_APPS += ('coursewarehistoryextended',)
+
+if ENV_TOKENS.get('API_ACCESS_MANAGER_EMAIL', None):
```

Comment:
No need for the if-statement. I'm fairly certain setting `API_ACCESS_MANAGER_EMAIL` to `None` will cause a great big error when we try to send the email. This seems more desirable than sending emails to a fake domain. In fact, we should probably drop use of the @example.com addresses in `common.py`.